### PR TITLE
File structure - Frontend

### PR DESCRIPTION
**Pull request by RazanTayyem**
```
  |__ public
  |     |__ index.html
  |    
  |__ src
  |     |__ app.js
  |     |__ index.js
  |     |__ components    
  |             |__ Button
  |             |      |__ index.js
  |             |      |__ index.css
  |             |
  |             |__ Form
  |             |      |__ index.js
  |             |      |__ index.css
  |             |
  |             |
  |             |__  Quiz
  |             |      |__ index.js
  |             |      |__ index.css
  |             |
  |             |__ Question
  |             |      |__ index.js
  |             |      |__ index.css
  |             |
  |             |__ Choice
  |             |      |__ index.js
  |             |      |__ index.css
  |             |
  |             |__ Image
  |             |      |__ index.js
  |             |      |__ index.css
  |             |
  |             |__ Pages
  |                     |                 
  |                     |__  Login 
  |                     |          |__ index.js
  |                     |          |__ index.css
  |                     |
  |                     |__ Signup 
  |                     |           |__ index.js
  |                     |           |__ index.css
  |                     |
  |                     |__  Start
  |                     |           |__ index.js
  |                     |           |__ index.css
  |                     |
  |                     |__  Result
  |                     |           |__ index.js
  |                     |           |__ index.css
  |                     |
  |                     |__   Activities
  |                     |           |__ index.js
  |                     |           |__ index.css
  |                     |
  |                     |__   Comparison
  |                                 |__ index.js
  |                                 |__ index.css
  |         
  |__ .eslintrc
  |__ .babelrc
  |__ .gitignore
  |__ package.json
  |__ package-lock.json
```

### CODE REVIEW

**Review by ameentalahmeh — APPROVED** [review]
seen :+1:

File: student/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "client",
```
Comment (mhmdtshref) [inline student/package.json]: Change this to `student`

**Review by mhmdtshref — CHANGES_REQUESTED** [review]
I think we'll use styled component, so rename the `index.css` files in the components folders to `index.style.js`.

File: student/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
+
+### Deployment
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
+
+### `npm run build` fails to minify
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
```
Comment (ghassanmas) [inline student/README.md]: No need for this auto generated file

**Review by ghassanmas — CHANGES_REQUESTED** [review]
just add proxy to the pacakge.json

**Pull request merged**